src/Libraries/DSIronPython/DSIronPython.csproj

```diff
@@ -36,6 +36,10 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="ICSharpCode.AvalonEdit">
+      <HintPath>..\..\..\extern\avalonEdit\ICSharpCode.AvalonEdit.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
```

Comment:
@mjkkirschner The IronPython SQL, Modules and WPF binaries are already added as references here to the DSIronPython project and the private flag is set to true. So the binaries are copied to the bin folder.